ArduCopter/events.cpp

```diff
@@ -17,7 +17,9 @@ void Copter::failsafe_radio_on_event()
         if (control_mode == AUTO && g.failsafe_throttle == FS_THR_ENABLED_CONTINUE_MISSION) {
             // continue mission
         } else if (control_mode == LAND && g.failsafe_battery_enabled == FS_BATT_LAND && failsafe.battery) {
-            // continue landing
+            // continue landing because we're already in the battery failsafe landing phase
+        } else if (control_mode == LAND && g2.fs_land_continue) {
+            // continue landing because FS_THR_LAND_CONT is true
```

Comment:
FS_THR_LAND_CONT -> FS_LAND_CONTINUE

Or maybe better, just *[...] because parameter is true*

Reply:
@Pedals2Paddles This wasn't fixed.